include/triton/Conversion/TritonGPUToLLVM/TargetInfoBase.h

```diff
@@ -106,6 +106,24 @@ class TargetInfoBase {
   virtual void localLoadOpAnnotation(triton::gpu::LocalLoadOp localLoadOp,
                                      Operation *llLoadOp) const {}
 
+  // -----------------------------------------------------------------------
```

Comment:
We probably want to move this to the top and also put `getClusterCTAId` (at L13 right now) in this section.